AirTote/Pages/TabChild/More.xaml.cs

```diff
@@ -0,0 +1,39 @@
+using AirTote.ViewModels;
+
+namespace AirTote.Pages.TabChild;
+
+public partial class More : ContentPage
+{
+
+
+	public More()
+	{
+		InitializeComponent();
+		
+	}
```

Comment:
意図した改行でしょうか?  この部分に空行を追加しても読みやすさには貢献しないように思えるため、削除をお願いします。

```suggestion
	public More()
	{
		InitializeComponent();
	}
```